views/index.html

```diff
@@ -17,56 +17,182 @@
     <!-- GOOGLE FONTS -->
     <link rel="preconnect" href="https://fonts.gstatic.com">
     <link href="https://fonts.googleapis.com/css2?family=Fredoka+One&display=swap" rel="stylesheet">
+
 </head>
 
 <body>
 
     <nav class="navbar navbar-expand-lg navbar-dark">
-        <a class="navbar-brand" href="/"><i class="fas fa-paw"></i></a>
```

Comment:
will this remove the paw print button's functionality?

Reply:
yes, I'll fix it right now.